docs/content/deployment/guides/kubernetes/how-to-migrate-your-instance.mdx

```diff
@@ -38,30 +68,73 @@ kubectl scale deploy $WEBSERVER_DEPLOYMENT_NAME --replicas=0
 kubectl scale deploy $DAEMON_DEPLOYMENT_NAME --replicas=0
 ```
 
-5. Run a Kubernetes Job with the `dagster instance migrate` command. You can check if the migration succeeded by inspecting the the resulting Pod of the Job.
+---
 
-```shell
-# Run `helm list` and save your Dagster Helm release name
-export HELM_DAGSTER_RELEASE_NAME=<DAGSTER_RELEASE_NAME>
-
-# The `helm template` command must be run from the directory containing the
-# `values.yaml` file you used to install the Dagster Helm chart.
-#
-# If needed, you can retrieve the currently applied `values.yaml` file
-# from the cluster by running:
-#
-# `helm get values $HELM_DAGSTER_RELEASE_NAME > values.yaml`
-#
-helm template $HELM_DAGSTER_RELEASE_NAME dagster/dagster \
-    --set "migrate.enabled=true" \
-    --show-only templates/job-instance-migrate.yaml \
-    --values values.yaml \
-    | kubectl apply -f -
-```
+## Step 3: Migrate the Dagster instance
+
+1. Run a Kubernetes job with the `dagster instance migrate` command. In the following example, `helm template` will run a job that uses this command and migrates the instance:
+
+   ```shell
+   # Run `helm list` and save your Dagster Helm release name
+   export HELM_DAGSTER_RELEASE_NAME=<DAGSTER_RELEASE_NAME>
+
+   # The `helm template` command must be run from the directory containing the
+   # `values.yaml` file you used to install the Dagster Helm chart.
+   #
+   # If needed, you can retrieve the currently applied `values.yaml` file
+   # from the cluster by running:
+   #
+   # `helm get values $HELM_DAGSTER_RELEASE_NAME > values.yaml`
+   #
+   helm template $HELM_DAGSTER_RELEASE_NAME dagster/dagster \
+       --set "migrate.enabled=true" \
+       --show-only templates/job-instance-migrate.yaml \
+       --values values.yaml \
+       | kubectl apply -f -
+   ```
+
+2. Next, you'll check that the migration succeeded. Run the following to get the name of the pod:
+
+   ```shell
+   kubectl get pods -l job-name=dagster-instance-migrate
+   ```
+
+3. Lastly, run the following to inspect the resulting pod, replacing `<POD_NAME>` with the name of the pod:
```

Comment:
Maybe add a "verify that the pod completed without any errors" so that they know what they are looking for